### ISSUE TITLE
Evaluate the quality of the predicted segmentation masks.

### ISSUE THREAD

**Issue by houzeyu2683** [junyanz/pytorch-CycleGAN-and-pix2pix#920]
In my case, I have a pair of images (img_real, img_fake), The channel is 3 (RGB). How to evaluate the quality of the predicted segmentation masks?

**Comment by junyanz** [comment]
Please see this [discussion](https://github.com/phillipi/pix2pix/issues/116).

**Comment by houzeyu2683** [comment]
I see the `evaluate.py` in the script folder, but I don't understand how to use it when I run the `train.py` and `test.py`.
The `evaluate.py` need the three folder, "cityscapes_dir", "result_dir" and "output_dir".
But after I run the `test.py`, the fake and real is group in the one folder since "--direction AtoB".
How to setup the arg that separate the generator image and ground truth image?
Thanks

**Comment by junyanz** [comment]
Please see this [post](https://github.com/phillipi/pix2pix/issues/112) for my answers. The program will produce the output images by default. It will be saved by different file names.

**Comment by houzeyu2683** [comment]
Should I convert model from pytorch to caffe and run evaluate.py in script folder?
